multinet/uploaders/csv.py

```diff
@@ -91,7 +99,10 @@ def upload(workspace: str, table: str) -> Any:
     # Read the request body into CSV format
     body = decode_data(request.data)
 
-    rows = list(csv.DictReader(StringIO(body)))
+    try:
+        rows = list(csv.DictReader(StringIO(body)))
+    except csv.Error:
+        raise CSVReadError()
```

Comment:
Do you have a file what would produce this error? Every time I try it raises other errors instead.

Reply:
I used a gpg-encrypted file, but I think you can use any binary file (like some random PDF file you have lying around) and it's likely to trigger this.

Reply:
Yep that did it, thanks :+1: